sections/featured-product.liquid

```diff
@@ -134,6 +155,45 @@
                     {%- endform -%}
                   </div>
                 {%- endif -%}
+              {%- when 'inventory' -%}
+                {%- if product.selected_or_first_available_variant.inventory_quantity != 0 -%}
+                  <p
+                    class="product__inventory no-js-hidden{% if block.settings.text_style == 'uppercase' %} caption-with-letter-spacing{% elsif block.settings.text_style == 'subtitle' %} subtitle{% endif %}"
+                    id="Inventory-{{ section.id }}"
+                    role="status"
+                    {{ block.shopify_attributes }}
+                  >
+                    {%- if product.selected_or_first_available_variant.inventory_quantity
+                        <= block.settings.inventory_threshold
+                    -%}
+                      <svg width="15" height="15">
```

Comment:
This is a decorative image, so we should hide it from screen readers with `role="presentation"`.

Reply:
Good call, Monica. Thanks.